pex/util.py

```diff
@@ -245,3 +245,15 @@ def iter_pth_paths(filename):
         if extras_dir_case_insensitive not in known_paths and os.path.exists(extras_dir):
           yield extras_dir
           known_paths.add(extras_dir_case_insensitive)
+
+
+def merge_split(*paths):
+  """
+  Merge paths into a single path delimited by colons and split on colons to return
```

Comment:
the docstring should start on the opening line like:

```
"""Docstring starts here.

...
"""
```

otherwise it won't appear correctly in IDEs/`help()`/etc.